test/recipes/30-test_evp_data/evpkdf_tls13_kdf.txt

```diff
@@ -5024,3 +5024,17 @@ Ctrl.data = hexdata:7c92f68bd5bf3638ea338a6494722e1b44127e1b7e8aad535f2322a644ff
 Ctrl.prefix = hexprefix:746c73313320
 Ctrl.label = hexlabel:6320652074726166666963
 Output = a8464234c7957b85460bf7abda8e20aa43b9e0944c02d76c1c28672619cf6978
+
+# Test that salt of arbitrary length works
+Availablein = fips
```

Comment:
Please remove this line - this should work in both default and fips providers.

Reply:
ooh this means available *only* in the fips provider ?

Reply:
> ooh this means available _only_ in the fips provider ?

Yes.